docs/core/tools/dotnet-install-script.md

```diff
@@ -22,34 +22,31 @@ ms.assetid: b64e7e6f-ffb4-4fc8-b43b-5731c89479c2
 
 macOS/Linux:
 
-`dotnet-install.sh [--channel] [--version] [--install-dir] [--architecture] [--shared-runtime] [--debug-symbols] [--dry-run] [--no-path] [--verbose] [--azure-feed] [--help]`
+`dotnet-install.sh [--channel] [--version] [--install-dir] [--architecture] [--shared-runtime] [--dry-run] [--no-path] [--azure-feed] [--verbose] [--help]`
 
 ## Description
 
-The `dotnet-install` scripts are used to perform a non-admin install of the CLI toolchain and the shared runtime. You can download the scripts from the [CLI GitHub repo](https://github.com/dotnet/cli/tree/rel/1.0.0/scripts/obtain). 
+The `dotnet-install` scripts are used to perform a non-admin install of the CLI toolchain and the shared runtime.
+
+It is recommended to use the stable version that is hosted on [.NET Core main website](https://dot.net). The direct path to the scripts are:
+
+* https://dot.net/v1/dotnet-install.sh (bash, UNIX)
+* https://dot.net/v1/dotnet-install.ps1 (powershell, Windows)
 
 The main usefulness of these scripts is in automation scenarios and non-admin installations. There are two scripts: One is a PowerShell script that works on Windows. The other script is a bash script that works on Linux/OS X. Both scripts have the same behavior. The bash script also reads PowerShell switches, so you can use PowerShell switches with the script on Linux/OS X systems. 
 
 The installation scripts download the ZIP/tarball file from the CLI build drops and proceed to install it in either the default location or in a location specified by `-InstallDir|--install-dir`. By default, the installation scripts download the SDK and install it. If you wish to only obtain the shared runtime, specify the `--shared-runtime` argument. 
 
 By default, the script adds the install location to the $PATH for the current session. Override this default behavior by specifying the `--no-path` argument. 
 
-Before running the script, install the required [dependencies](https://github.com/dotnet/core/blob/master/Documentation/prereqs.md).
-
-You can install a specific version using the `--version` argument. The version must be specified as a 3-part version (for example, 1.0.0-13232). If omitted, it defaults to the first [global.json](global-json.md) file found in the hierarchy above the folder where the script is invoked that contains the `version` property. If that isn't present, it will use the latest version.
-
-You can also use this script to obtain the SDK or shared runtime debug binaries with debug symbols by using the `--debug` argument. If you fail to do this on first install and realize later that you need the debug symbols, you can re-run the script with the `--debug` argument and the SDK version you installed to obtain the debug symbols. 
+You can install a specific version using the `--version` argument. The version must be specified as a 3-part version (for example, 1.0.0-13232). If omitted, it will use the `latest` version.
```

Comment:
no prereqs needed any longer? not even for older versions?

will use -> uses

Reply:
prereqs section put back in

Reply:
cool. are they updated for 2.0? /cc @Petermarcu re. this link: https://github.com/dotnet/core/blob/master/Documentation/prereqs.md